src/helpers/figures/charts/chart_common_line_scatter.ts

```diff
@@ -279,7 +279,7 @@ export function createLineOrScatterChartRuntime(
       });
     }
 
-    const backgroundColor = rgbaToHex(backgroundRGBA);
+    const backgroundColor = setColorAlpha(rgbaToHex(backgroundRGBA), 0.3);
```

Comment:
what are those changes for? Why do the background color of the line/scatter charts have to change for radar charts ? And why does line/scatter charts datasets have a background color in the first place, if these are not area chart